cmd/server/install.go

```diff
@@ -112,11 +112,9 @@ func NewInstallCmd() *cobra.Command {
 			}
 			baseLink := fmt.Sprintf("http://127.0.0.1:%v", port)
 
-			// todo call hook on install finished
-			// apiLink := fmt.Sprintf("%s/api/v2", baseLink)
-			// if err := auth.Login("", apiLink); err != nil {
-			// 	return err
-			// }
+			if onInstalled != nil {
+				onInstalled(baseLink)
+			}
```

Comment:
It may be a good idea to get this code out of the installation since its not really generic function
I re command to auth in the CLI